projects/plugins/jetpack/changelog/enhance-block-themes-hide-customizer

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+The Customize menu is removed on self hosted sites that are using a block based theme.
```

Comment:
Nitpick: This message will be visible in the public changelog of the next Jetpack release. Since this change will affect Jetpack users, it'd be good to use a more clear language since they might don't know what a "self hosted site" is. I think we should also specify that this change only affects the WordPress.com menu (and not the WP Admin menu).

Maybe something like this?

```suggestion
Removed the "Appearance > Customize" entry from the WordPress.com menu when the site uses a block-based theme.
```

Reply:
Sure